solr/solr-ref-guide/modules/indexing-guide/pages/post-tool.adoc

```diff
@@ -44,51 +44,59 @@ The basic usage of `bin/solr post` is:
-
-OPTIONS
-=======
-  Solr options:
-    -url <base Solr update URL>
-    -commit issue a commit
-    -u or -credentials <user:pass> (sets BasicAuth credentials)
-
-  Web crawl options:
-    -recursive <depth> (default: 1)
-    -delay <seconds> (default: 10)
-
-  Directory crawl options:
-    -delay <seconds> (default: 0)
-
-  stdin/args options:
-    -type <content/type> (default: application/json)
-
-
-  Other options:
-    -filetypes <type>[,<type>,...] (default: xml,json,csv,pdf,doc,docx,ppt,pptx,xls,xlsx,odt,odp,ods,ott,otp,ots,rtf,htm,html,txt,log)
-    -params "<key>=<value>[&<key>=<value>...]" (values must be URL-encoded; these pass through to Solr update request)
-    -out output the Solr responses to console
-    -format solr (sends application/json content as Solr commands to /update instead of /update/json/docs
-    
-Examples:
-
-* JSON file: bin/solr post -url http://localhost:8983/wizbang/update events.json
-* XML files: bin/solr post -url http://localhost:8983/records/update article*.xml
-* CSV file: bin/solr post -url http://localhost:8983/signals/update LATEST-signals.csv
-* Directory of files: bin/solr post -filetypes xml,json,csv -url http://localhost:8983/myfiles/update ~/Documents
-* Web crawl: bin/solr post -mode web -url http://localhost:8983/gettingstarted/update -recursive 1 -delay 1 https://solr.apache.org/ 
-* Standard input (stdin): echo '{commit: {}}' | bin/solr post -mode stdin -url http://localhost:8983/my_collection/update -out
-* Data as string: bin/solr post -url http://localhost:8983/signals/update -mode args -type text/csv -out $'id,value\n1,0.47' 
-
+usage: post
+ -c,--name <NAME>                             Name of the collection.
+ -delay <delay>                               If recursive then delay will
+                                              be the wait time between
+                                              posts.  default: 10 for web,
+                                              0 for files
+ -filetypes <<type>[,<type>,...]>             default:
+                                              xml,json,jsonl,csv,pdf,doc,d
+                                              ocx,ppt,pptx,xls,xlsx,odt,od
+                                              p,ods,ott,otp,ots,rtf,htm,ht
+                                              ml,txt,log
+ -format                                      sends application/json
+                                              content as Solr commands to
+                                              /update instead of
+                                              /update/json/docs
+ -help                                        Print this message
+ -mode <mode>                                 Files crawls files, web
+                                              crawls website. default:
+                                              files.
+ -skipcommit                                  Skip committing newly
+                                              posted documents
```

Comment:
Given the audience of new users, maybe explain what the ramifications of that is.  Can be brief:  `Do not "commit", and thus changes won't be visible.  Commits may happen later, perhaps via autoSoftCommit configuration.`

Reply:
reworked it a bit.